pkg/cmd/server/admin/create_servercert.go

```diff
@@ -113,10 +129,16 @@ func (o CreateServerCertOptions) CreateServerCert() (*crypto.TLSCertificateConfi
 
 	var ca *crypto.TLSCertificateConfig
 	written := true
+
+	expirationDate, err := time.Parse(time.RFC3339, o.ExpirationDate)
+	if err != nil {
```

Comment:
For now, I'm parsing a date twice -- first, during validation, and, second, here, right before using. Didn't find a better solution for that yet.